[2021-05-25 20:56:17,687] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T19:53:07.052122+00:00 [queued]>
[2021-05-25 20:56:17,840] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T19:53:07.052122+00:00 [queued]>
[2021-05-25 20:56:17,841] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 20:56:17,841] {taskinstance.py:1043} INFO - Starting attempt 2 of 2
[2021-05-25 20:56:17,847] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 20:56:18,033] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): Get_COVID19_Cases_For_Jordan_2020ToNow> on 2021-05-25T19:53:07.052122+00:00
[2021-05-25 20:56:18,048] {standard_task_runner.py:52} INFO - Started process 129 to run task
[2021-05-25 20:56:18,073] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'covid19_cases_Jordan', 'Get_COVID19_Cases_For_Jordan_2020ToNow', '2021-05-25T19:53:07.052122+00:00', '--job-id', '107', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/workflow.py', '--cfg-path', '/tmp/tmpzbekpbg6', '--error-file', '/tmp/tmp856dfqnz']
[2021-05-25 20:56:18,096] {standard_task_runner.py:77} INFO - Job 107: Subtask Get_COVID19_Cases_For_Jordan_2020ToNow
[2021-05-25 20:56:18,456] {logging_mixin.py:104} INFO - Running <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T19:53:07.052122+00:00 [running]> on host 4e0af5d83701
[2021-05-25 20:56:18,591] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=waed
AIRFLOW_CTX_DAG_ID=covid19_cases_Jordan
AIRFLOW_CTX_TASK_ID=Get_COVID19_Cases_For_Jordan_2020ToNow
AIRFLOW_CTX_EXECUTION_DATE=2021-05-25T19:53:07.052122+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-25T19:53:07.052122+00:00
[2021-05-25 21:21:42,608] {logging_mixin.py:104} INFO - It took 25.4 minutes to get all data
[2021-05-25 21:21:42,980] {python.py:118} INFO - Done. Returned value was: None
[2021-05-25 21:21:43,063] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=covid19_cases_Jordan, task_id=Get_COVID19_Cases_For_Jordan_2020ToNow, execution_date=20210525T195307, start_date=20210525T205617, end_date=20210525T212143
[2021-05-25 21:21:43,485] {local_task_job.py:188} WARNING - State of this instance has been externally set to success. Terminating instance.
[2021-05-25 21:21:43,523] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 129
[2021-05-25 21:21:43,837] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
[2021-05-25 21:21:43,938] {process_utils.py:66} INFO - Process psutil.Process(pid=129, status='terminated', exitcode=1, started='20:56:17') (129) terminated with exit code 1
